### PR TITLE
docs(configure-mayastor): updated note to bypath or byid

### PR DESCRIPTION
**Pull request by AJDatacore**
In reference to this issue: https://github.com/openebs/mayastor/issues/1530

Hi @tiagolobocastro Could you please confirm if this change is relevant? If now, can you suggest the right place to add this note?

### CODE REVIEW

File: quickstart/configure-mayastor.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Mayastor versions before 2.0.1 had an upper limit on the number of retry attempt
 #### Permissible Schemes for `spec.disks` under DiskPool CR
 
 {% hint style="info" %}
-It is highly recommended to specify the disk using a unique device link that remains unaltered across node reboots. One such device link is its `UUID`.
+It is highly recommended to specify the disk using a unique device link that remains unaltered across node reboots. You can create this disk by-path or by-id. One such device link is its `UUID`.
```
Comment (tiagolobocastro) [inline quickstart/configure-mayastor.md]:
```suggestion
It is highly recommended to specify the disk using a unique device link that remains unaltered across node reboots. Examples of such device links are: by-path or by-id.
```

File: quickstart/configure-mayastor.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Mayastor versions before 2.0.1 had an upper limit on the number of retry attempt
...
 To get the UUID of a disk, execute:
 `sudo blkid`
```
Comment (tiagolobocastro) [inline quickstart/configure-mayastor.md]:
```suggestion
Easy way to retrieve device link for a given node:
> kubectl mayastor get block-devices worker
DEVNAME       DEVTYPE  SIZE      AVAILABLE  MODEL                             DEVPATH                                                           MAJOR  MINOR  DEVLINKS 
 /dev/nvme0n1  disk     894.3GiB  yes        Dell DC NVMe PE8010 RI U.2 960GB  /devices/pci0000:30/0000:30:02.0/0000:31:00.0/nvme/nvme0/nvme0n1  259    0      "/dev/disk/by-id/nvme-eui.ace42e00164f0290", "/dev/disk/by-path/pci-0000:31:00.0-nvme-1", "/dev/disk/by-dname/nvme0n1", "/dev/disk/by-id/nvme-Dell_DC_NVMe_PE8010_RI_U.2_960GB_SDA9N7266I110A814"
```

Comment (tiagolobocastro) [reply quickstart/configure-mayastor.md]: Also on line 28 (can't add comment there):
change by-uuid with by-id

**Review by tiagolobocastro — APPROVED** [review]

**Review by blaisedias — APPROVED** [review]

File: quickstart/configure-mayastor.md [hunk 3]
```diff
@@ -16,16 +16,22 @@ Mayastor versions before 2.0.1 had an upper limit on the number of retry attempt
...
+/dev/nvme0n1  disk     894.3GiB  yes        Dell DC NVMe PE8010 RI U.2 960GB  /devices/pci0000:30/0000:30:02.0/0000:31:00.0/nvme/nvme0/nvme0n1  259    0      "/dev/disk/by-id/nvme-eui.ace42e00164f0290", "/dev/disk/by-path/pci-0000:31:00.0-nvme-1", "/dev/disk/by-dname/nvme0n1", "/dev/disk/by-id/nvme-Dell_DC_NVMe_PE8010_RI_U.2_960GB_SDA9N7266I110A814"
+```
 
 Usage of the device name (for example, /dev/sdx) is not advised, as it may change if the node reboots, which might cause data corruption.
 {% endhint %}
 
 | Type | Format | Example |
 | :--- | :--- | :--- |
-| Disk(non PCI) with disk-by-guid reference <i><b>(Best Practice)</b></i> | Device File | aio:////dev/disk/by-uuid/<uuid> OR uring:////dev/disk/by-uuid/<uuid> |
+| Disk(non PCI) with disk-by-guid reference <i><b>(Best Practice)</b></i> | Device File | aio:////dev/disk/by-id/<id> OR uring:////dev/disk/by-id/<id> |
```
Comment (tiagolobocastro) [inline quickstart/configure-mayastor.md]: sorry one more nit, there's an extra slash here
```suggestion
| Disk(non PCI) with disk-by-guid reference <i><b>(Best Practice)</b></i> | Device File | aio:///dev/disk/by-id/<id> OR uring:///dev/disk/by-id/<id> |
```

**Review by tiagolobocastro — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
~Thanks @AJDatacore, could you please cherry-pick to develop as well?~

cherry-picked

**Pull request merged**